tests/core.c

```diff
@@ -252,6 +257,19 @@ void test_sepby(void) {
   PT_ASSERT(mpc_test_pass(CommaSepIdent, "one,two,three", "onetwothree", streq, free, strprint));
 
   mpc_delete(CommaSepIdent);
+
+  mpc_parser_t* CommaSepIdent1 = mpc_sepby1(fold_vals, mpc_char(','), mpc_ident());
+  mpc_result_t r;
+
+  mpc_parse("<test>", "uno,dos,tres,cuatro", CommaSepIdent1, &r);
+  char **vals = r.output;
+  PT_ASSERT(strcmp("uno",    vals[0]) == 0);
+  PT_ASSERT(strcmp("dos",    vals[1]) == 0);
+  PT_ASSERT(strcmp("tres",   vals[2]) == 0);
+  PT_ASSERT(strcmp("cuatro", vals[3]) == 0);

```

Comment:
This is the main motivation, it wasn't possible to do this with the previous `sepby1`.